docs/markdown/Gnome-module.md

```diff
@@ -40,6 +40,7 @@ takes two positional arguments. The first one is the name of the
 resource and the second is the XML file containing the resource
 definitions. If the name is `foobar`, Meson will generate a header
 file called `foobar.h`, which you can then include in your sources.
+The resources specified are automatically added as dependencies.
```

Comment:
Perhaps saying that they are added "as dependencies of the generated target" would add a little more clarity.

Reply:
Done, thanks for the input.